x/storage/types/keys.go

```diff
@@ -101,6 +112,12 @@ func GetObjectByIDKey(objectId math.Uint) []byte {
 	return append(ObjectByIDPrefix, seq.EncodeSequence(objectId)...)
 }
 
+// GetShadowObjectByIDKey return the shadowObjectId store key
+func GetShadowObjectByIDKey(objectId math.Uint) []byte {
```

Comment:
We don't need `bucketName + objectName` -> `id` -> `shadow object`.
We can just `bucketName + objectName` -> `shadow object`, to save storage and make code easier.

Reply:
Yep, change is applied.